views/js/runner/plugins/tools/highlighter/highlighter.js

```diff
@@ -89,6 +112,22 @@ define([
          */
         return eventifier({
 
+            enable: function enable() {
```

Comment:
Could you please add the jsdoc comment (even if it is artificial, it's more for the consistency)